pkg/functions/deploy.go

```diff
@@ -73,6 +73,45 @@ func RunDeploy(w io.Writer, sdk *SdkStatus) error {
 			if err != nil {
 				return err
 			}
+		} else if step.Build.Builder != "" {
+			if step.Build.Destination == "" {
+				return fmt.Errorf("Please provide a destination for the build image.")
+			}
+			if step.Build.MainPath == "" {
+				return fmt.Errorf("Please provide the relative path to the main file.")
+			}
+			packCommand := fmt.Sprintf("pack build %s --builder %s --env GOOGLE_BUILDABLE=%s --publish", step.Build.Destination, step.Build.Builder, step.Build.MainPath)
```

Comment:
this should use exec.LookPath to find the pack binary, rather than running /bin/sh (which may not be installed on the host)

Reply:
Ah I was wondering what that did. Done!